pdns/dnsdistdist/dnsdist-lua-inspection.cc

```diff
@@ -1048,7 +1048,14 @@ void setupLuaInspection(LuaContext& luaCtx)
   /* DynBlock object accessors */
   luaCtx.registerMember("reason", &DynBlock::reason);
   luaCtx.registerMember("domain", &DynBlock::domain);
-  luaCtx.registerMember("until", &DynBlock::until);
+  luaCtx.registerMember<DynBlock, timespec>(
+    "until", [](const DynBlock& block) {
+      timespec nowMonotonic{};
+      gettime(&nowMonotonic);
+      timespec nowRealTime{};
+      gettime(&nowRealTime, true);
+      nowRealTime.tv_sec += (block.until.tv_sec - nowMonotonic.tv_sec);
```

Comment:
I'd rather do the `nowRealTime` computation using the full timespecs, and only then convert to secs. e.g. `DiffTime` and a `round()`?

Reply:
Grumble, bit of course I always forget that there is no standardized timespec computation API

Reply:
> I'd rather do the `nowRealTime` computation using the full timespecs. Because the nsecs can work agains you I think.

We now do so, please review carefully!